server/src/database/migrations/00003-add-constraints.ts

```diff
@@ -9,16 +9,6 @@ export default {
   up: async (queryInterface: QueryInterface): Promise<void> => {
     const transaction: Transaction = await queryInterface.sequelize.transaction();
     try {
-      await queryInterface.addConstraint('course', {
```

Comment:
This constraint should be added for course instances now.

Reply:
Agree. Even tough we have yup validating this in the backend, I would still have the database level check also.

Reply:
Agree. Even tough we have yup validating this in the backend, I would still have the database level check also.

Reply:
True, have probably missed this.